### PR TITLE
feat(charts)!: refactor of evm chart

### PR DESCRIPTION
**Pull request by joroshiba**
## Summary
Large refactor of the EVM chart which break out the various components, and restructures the value file to have the genesis clearly defined. The evm chart itself becomes only `geth`

* `conductor.` New charts for `composer` & `evm-faucet` are added. A new `evm-stack` chart is created which allows user to run many of these components similar to how EVM chart functioned before, it is migrated to using the blockscout published chart instead of our own chart for blockscout (to be tested still).

## Background
The EVM chart needed to be split up for a long time, the blockscout and faucet dependencies mostly useful for local dev. Composer split out helps at scale.

### CODE REVIEW

File: charts/evm-rollup/values.yaml
Comment (joroshiba) [inline charts/evm-rollup/values.yaml:111]: this allows any of the flags on geth to be set when running the chart, for them to have a value or not, and to set a `condition` which can be executed in an helm `if` statement for whether or not to set the flag.

File: charts/evm-rollup/values.yaml
Comment (joroshiba) [inline charts/evm-rollup/values.yaml:24]: moved genesis params into a central location instead of being split up across regions

File: charts/evm-rollup/values.yaml
Comment (joroshiba) [inline charts/evm-rollup/values.yaml:207]: moved metrics, otel, service monitor to top level, matches other charts closer

File: charts/evm-stack/Chart.yaml
Comment (joroshiba) [inline charts/evm-stack/Chart.yaml:20]: This stack chart is just a wrapper for all the other charts you might want to run.

File: dev/values/rollup/dev.yaml
Comment (joroshiba) [inline dev/values/rollup/dev.yaml:1]: this file utilizes the evm-stack wrapper chart

File: charts/deploy.just
Comment (joroshiba) [inline charts/deploy.just:169]: this was done just for testing purposes, should be changed to disable all before merge.

File: charts/composer/Chart.yaml
Comment (joroshiba) [inline charts/composer/Chart.yaml:1]: general note that this does not currently support ingress for grpc, but we have a service so we could expose it seperately

File: charts/composer/templates/_helpers.tpl
Comment (quasystaty1) [inline charts/composer/templates/_helpers.tpl:26]: nit: newfile

File: charts/composer/templates/service.yaml
Comment (quasystaty1) [inline charts/composer/templates/service.yaml:29]: nit: newline

File: charts/deploy.just
Comment (quasystaty1) [reply charts/deploy.just:169]: Why disabling blockscout/faucet by default?

File: charts/evm-faucet/Chart.yaml
Comment (quasystaty1) [inline charts/evm-faucet/Chart.yaml:33]: nit: Maybe add `quasystaty1` as maintainers for consistency with other charts?

File: charts/evm-rollup/values.yaml
Comment (quasystaty1) [inline charts/evm-rollup/values.yaml:104]: Should this be configurable for each component separately?

File: dev/values/rollup/dev.yaml
Comment (quasystaty1) [inline dev/values/rollup/dev.yaml:170]: default name is `astria` not `flame`
```suggestion
    rollupName: astria
```

**Review by quasystaty1 — APPROVED** [review]

**Pull request merged**
